src/System.Memory/ref/System.Memory.cs

```diff
@@ -278,9 +278,9 @@ public abstract partial class OwnedMemory<T> : System.Buffers.IRetainable, Syste
     }
     public static partial class BuffersExtensions
     {
-        public static void CopyTo<T>(this System.Buffers.ReadOnlySequence<T> sequence, System.Span<T> destination) { }
-        public static System.Nullable<System.SequencePosition> PositionOf<T>(this System.Buffers.ReadOnlySequence<T> sequence, T value) where T : System.IEquatable<T> { throw null; }
-        public static T[] ToArray<T>(this System.Buffers.ReadOnlySequence<T> sequence) { throw null; }
+        public static void CopyTo<T>(in this System.Buffers.ReadOnlySequence<T> sequence, System.Span<T> destination) { }
```

Comment:
Let's not change public API in this PR. Why is this change from `this` to `in`

Reply:
Its a 64 byte (256 bit) struct; with four fields 2 of which are objects

Reply:
Its a 64 byte struct; with four fields 2 of which are objects; being passed with a 16 byte struct, 1 object; so the parameters to this method are 80 bytes 3 of which are objects.

Passing `in` cuts it to 8 bytes + 16 bytes

Reply:
This relates to https://github.com/dotnet/coreclr/issues/16619

From previous discussion on Spans - https://github.com/dotnet/corefx/issues/27238
> I would prefer we pass small structs by-value (implicitly converted to by-ref in some ABIs) as this gives us the best opportunity to optimize down the road, and an additional incentive to do so.
Staying with by-value would mean sacrificing some perf in the meantime -- most of the "selectively jit better" technologies are in early stages of development.

So, the conclusion was to avoid using `in` for small structs and let the JIT improve.

Since this isn't a small struct, is this a good use of `in` here? Can we get a general guidance on when we should use `in` based on struct sizes? Could this be something that the JIT can handle automagically instead of us updating the API surface?

cc @AndyAyersMS, @CarolEidt

Reply:
We decided in API Review that `in` is fine for pref sensitive methods that take structs larger than two longs.

Reply:
Using `in` for structs larger than 16 bytes seems reasonable.